<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyEditorEBDClass</class>
 <widget class="QMainWindow" name="MyEditorEBDClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1357</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyEditorEBD</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/needle.png</normaloff>Resources/needle.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="DesignContainer" name="designContainer" native="true">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1357</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionTile_Window"/>
    <addaction name="actionCascade_Window"/>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>File</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="actionSave_File"/>
   <addaction name="actionOpen_File"/>
   <addaction name="separator"/>
   <addaction name="actionCCTable"/>
  </widget>
  <widget class="QToolBar" name="colorToolBar">
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>Color palette</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="cursorToolBar">
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>Design modify</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionTransform"/>
   <addaction name="actionEditStitches"/>
   <addaction name="separator"/>
   <addaction name="actionRotate"/>
   <addaction name="actionScale"/>
   <addaction name="separator"/>
   <addaction name="actionPointing"/>
   <addaction name="actionDesignGrid"/>
   <addaction name="actionAxis"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dkImageMap">
   <property name="minimumSize">
    <size>
     <width>241</width>
     <height>38</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Image Map</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="WidgetNavigator" name="wdNavigator" native="true">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dkSequenceViewer">
   <property name="minimumSize">
    <size>
     <width>241</width>
     <height>38</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Sequence Viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="wdSequenceViewer" native="true">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dkHeader">
   <property name="windowTitle">
    <string>Header Info</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="WidgetHeaderInfo" name="wdHeaderInfo" native="true">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dkPoint">
   <property name="windowTitle">
    <string>Points</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="WidgetPointInfo" name="wdPoint" native="true">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dkCCTable">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="windowTitle">
    <string>C/C Table</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="WidgetCCTable" name="wdCCTable" native="true">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dkColorFilm">
   <property name="windowTitle">
    <string>Color Film Table</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QGridLayout" name="gridLayout_7">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="WidgetColorFilm" name="wdColorFilm" native="true">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dkDesign">
   <property name="windowTitle">
    <string>Design Navigator</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QGridLayout" name="gridLayout_8">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="WidgetDesignNavigator" name="wdDesign" native="true">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/open_dir.png</normaloff>:/MyEditorEBD/Resources/open_dir.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/save.png</normaloff>:/MyEditorEBD/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/exit.png</normaloff>:/MyEditorEBD/Resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTransform">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/cursor.png</normaloff>:/MyEditorEBD/Resources/cursor.png</iconset>
   </property>
   <property name="text">
    <string>actionTransform</string>
   </property>
   <property name="toolTip">
    <string>Re-arrange objects</string>
   </property>
  </action>
  <action name="actionEditStitches">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/edit.png</normaloff>:/MyEditorEBD/Resources/edit.png</iconset>
   </property>
   <property name="text">
    <string>actionEditStitches</string>
   </property>
   <property name="toolTip">
    <string>Edit stitches of objects</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/rotate.png</normaloff>:/MyEditorEBD/Resources/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate design</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/scale.png</normaloff>:/MyEditorEBD/Resources/scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
   <property name="toolTip">
    <string>Scale design</string>
   </property>
  </action>
  <action name="actionDesignGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/grid.png</normaloff>:/MyEditorEBD/Resources/grid.png</iconset>
   </property>
   <property name="text">
    <string>DesignGrid</string>
   </property>
   <property name="toolTip">
    <string>Design grid is visible</string>
   </property>
  </action>
  <action name="actionAxis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/coordinate.png</normaloff>:/MyEditorEBD/Resources/coordinate.png</iconset>
   </property>
   <property name="text">
    <string>Axis</string>
   </property>
   <property name="toolTip">
    <string>Axis is visble</string>
   </property>
  </action>
  <action name="actionPointing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/pointing.png</normaloff>:/MyEditorEBD/Resources/pointing.png</iconset>
   </property>
   <property name="text">
    <string>Pointing</string>
   </property>
  </action>
  <action name="actionTile_Window">
   <property name="text">
    <string>Tile Window</string>
   </property>
  </action>
  <action name="actionCascade_Window">
   <property name="text">
    <string>Cascade Window</string>
   </property>
  </action>
  <action name="actionCCTable">
   <property name="icon">
    <iconset resource="MyEditorEBD.qrc">
     <normaloff>:/MyEditorEBD/Resources/cctable.png</normaloff>:/MyEditorEBD/Resources/cctable.png</iconset>
   </property>
   <property name="text">
    <string>CCTable</string>
   </property>
   <property name="toolTip">
    <string>CCTable</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DesignContainer</class>
   <extends>QWidget</extends>
   <header>designcontainer.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetNavigator</class>
   <extends>QWidget</extends>
   <header>WidgetNavigator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetHeaderInfo</class>
   <extends>QWidget</extends>
   <header>WidgetHeaderInfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPointInfo</class>
   <extends>QWidget</extends>
   <header>WidgetPointInfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCCTable</class>
   <extends>QWidget</extends>
   <header>WidgetCCTable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetColorFilm</class>
   <extends>QWidget</extends>
   <header>WidgetColorFilm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetDesignNavigator</class>
   <extends>QWidget</extends>
   <header>WidgetDesignNavigator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyEditorEBD.qrc"/>
 </resources>
 <connections/>
</ui>
